src/components/2FA/Passphrase2FA.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 import Modal, { ModalContent } from 'cozy-ui/react/Modal'
+import Input from 'cozy-ui/react/Input'
```

Comment:
Do settings has an alias somewhere to use the transpiled version?

Reply:
Yes there's one ;)